wrap/gtwrap/pybind_wrapper.py

```diff
@@ -292,6 +296,37 @@ def wrap_namespace(self, namespace):
             )
         return wrapped, includes
 
+    def wrap_pickle(self, cpp_class):
+        """Wrap to enable pickling on the generated Python class.
+        
+        Pickling is supported by reusing the serialization functionality already build in to many GTSAM types.
+
+        Ref: https://pybind11.readthedocs.io/en/stable/advanced/classes.html#pickling-support
+        """
+        if cpp_class not in [
+            "gtsam::Cal3Bundler",
+            "gtsam::PinholeCameraCal3Bundler", 
+            "gtsam::Pose3", 
+            "gtsam::Rot3", 
+            "gtsam::SfmTrack"
+            "gtsam::Unit3", 
+        ]:
+            # TODO: fix this check by just checking if serialization is supported.
```

Comment:
I am relying on the serialization function to make my life easier. Things like `BetweenFactor` do not support the serialization functionality. I know this check against a list of 5 objects (what is needed for GT-SFM right now) is not optimal, and I request alternate suggestions.

Reply:
Is it correct to just return an empty string for these 6 classes?

Reply:
This won't apply now. With help from @ProfFan, we now add pickle support bundled with serialization. So all classes which can be serialized are supported now.